storage/nullifiers.go

```diff
@@ -0,0 +1,56 @@
+package storage
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"go.vocdoni.io/dvote/db"
+	"go.vocdoni.io/dvote/db/prefixeddb"
+)
+
+func (s *Storage) lockNullifier(nullifier *big.Int) error {
```

Comment:
I'd use a memory map here (or a sync.map), I can only see advantages.
1. Faster
2. No garbage collected on restarts or unexpected errors
3. No need to take locks that would slow-down other operations
4. Simpler

However, on start, you must fill the sync.Map with the current pending queues (verifiedvotes + aggregated). The storage package already have some init functions that clean stale reservations and so on. Thus you can do something like that.